UploadDaemon/Configuration/GlobPatternList.cs

```diff
@@ -75,5 +75,17 @@ public Boolean Matches(string text)
         {
             return includeRegexes.Any(regex => regex.IsMatch(text)) && !excludeRegexes.Any(regex => regex.IsMatch(text));
         }
+
+        /// <inheritdoc/>
+        public override bool Equals(object obj)
+        {
+            return this.Describe().Equals((obj as GlobPatternList)?.Describe());
```

Comment:
that seems hacky. I wouldn't expect that changing the describe method influences equals, so this is a chance for very subtle bugs. Can you please change to a proper eqauls method?